### DISCUSSION TITLE
Log in issues? Getting a "server error" and am unable to login

### DISCUSSION THREAD

**Discussion by bfencken** [bookwyrm-social/bookwyrm#3243] [Q&A]
Tried resetting password, and still nothing. Kind of a generic error message.  Just me??
![image](https://github.com/bookwyrm-social/bookwyrm/assets/54144371/4227bba8-e660-488c-b5c2-c285c37f8fdf)

**Comment by mouse-reeve** [comment]
I'm closing this as a duplicate of #3271
